docs/key-issues/cross_platform.md

```diff
@@ -0,0 +1,33 @@
+# Cross-platform installation
+
+The historical assumption of compilation is that the platform where the code is
+compiled will be the same as the platform where the final code will be executed
+(if not literally the same machine, then at least one that is CPU and ABI
+compatible at the operating system level). This is a reasonable assumption for
+most desktop projects; However, for mobile platforms, this isn't the case.
+
+On mobile platforms, an app is compiled on a desktop platform, and transferred
+to the mobile device (or a simulator) for testing. The compiler is not executed
+on device. Therefore, it must be possible to build a binary artefact for a CPU
+architecture and a ABI that is different the platform that is running the
+compiler.
+
```

Comment:
Embedded devices like the Raspberry Pi (`armv7` in particular) often need cross-compiling too. That's a prominent enough case to mention here I think (also xref https://www.piwheels.org/ for that).

And in general, cross-compiling to architectures that are not/less available in free CI (`ppc64le`, `s390x`) would be nice to have as well

Reply:
Good point - I'll add a note about this.